vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+author: Kaitlyn Johnson
+output:
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Mathematical methods for baselinenowcast}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# `baselinenowcast` mathematical model
```

Comment:
pedantic point but dont really need the package name in the docs of the package?

Reply:
Removed